src/app/users/filters.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import { Button, Flex, TextField, Select, Link, Checkbox, Text } from '@radix-ui/themes';
+import { useTranslations } from 'next-intl';
+import { MixerHorizontalIcon, PlusIcon } from '@radix-ui/react-icons';
+
+interface Filters {
+  roleType?: string;
+  searchQuery?: string;
+  showDeleted?: boolean;
+}
+
+interface FiltersProps {
+  filters: Filters;
+  onFiltersChange: (filters: Filters) => void;
+  isAdmin: boolean;
```

Comment:
Can we change this to `showDeleted` instead? That way the business logic of who gets to see deleted users isn't baked into the form component.